README.md

```diff
@@ -1 +1,19 @@
-# config-examples
\ No newline at end of file
+# config-examples
+
+## Kubernetes Deployments Examples
+
+- [API Docker image](https://hub.docker.com/r/screwdrivercd/screwdriver)
+- [UI Docker image](https://hub.docker.com/r/screwdrivercd/ui)
+- [Artifact Store Docker image](https://hub.docker.com/r/screwdrivercd/store)
+
+### Usage
+
+Describe current status:
+```
+$ kubectl describe -f FILENAME
+```
+
+Update deployment:
```

Comment:
This can also be used to create a new deployment. Will `Create or update` be better?